resources/assets/components/Signup/index.js

```diff
@@ -262,6 +263,34 @@ class Signup extends React.Component {
     }
   }
 
+  // Delete a signup.
+  deleteSignup(signupId, event) {
+    event.preventDefault();
+    const confirmed = confirm('🚨🔥🚨 Are you sure you want to delete this? 🚨🔥🚨');
+
+    if (confirmed) {
+      // Make API request to Rogue to delete the signup on the backend.
+      const authorizedRequest = new RestApiClient(window.location.origin, {
+        headers: { 'Authorization' : `Bearer ${window.AUTH}`}
+      });
+
+      const response = authorizedRequest.delete('api/v3/signups/'.concat(signupId));
```

Comment:
This is subjective, but this might read a little better with a template tag:

```js
const response = authorizedRequest.delete(`api/v3/signups/${signupId}`);
```